### ISSUE TITLE
When using banners in the index page, Page Down doesn't work when the page is exactly full

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#1023]
Original [issue 1024](https://code.google.com/p/moviejukebox/issues/detail?id=1024) created by Omertron on 2009-09-28T06:25:56.000Z:

I'm using r1111.

If I'm using 8 banners-per-page in the index and I have, for example, 16 TV 
series (grouped into sets), then when I hit Page Down on the 2nd page, it 
will _not_ bring me back to page 1.

According to Other_TV Shows_2.xml, 
library/category[@current='true']/index[@current='true']/@next is set to 
Other_TV Shows_2, when it should be set to Other_TV Shows_1. So it 
basically keeps me on page 2.

Now if I have only 15 TV series (grouped into sets), Page Down works as 
expected and brings me back to page 1. 14 TV series works also, all the way 
down to 9 TV series.

This goes for 3 pages (24 TV series) also -- it will keep me on page 3 when 
I press Page Down. However, 23 TV series will bring me back to page 1 when 
I press Page Down.

So basically, when the page is exactly full (all &quot;cells&quot; assigned a TV 
series), Page Down (@next) isn't doing its job.
